addons/sourcemod/scripting/nt_classlimit.sp

```diff
@@ -122,11 +122,40 @@ public void OnPluginStart()
 		HookClassSelectionPfns(client);
 		break;
 	}
-
+	
+	RegAdminCmd("sm_classlimit", Command_Limit, ADMFLAG_GENERIC);
+	
 	// Create the default config file, if it doesn't exist yet
 	AutoExecConfig();
 }
 
+public Action Command_Limit(int client, int args)
+{
+	if (args == 0)
+	{
+		ReplyToCommand(client, "Usage !classlimit x where x sets the limit for all classes");
+		return Plugin_Handled;
+	}
+	
+	char limitArg[2 + 1];
+	GetCmdArg(1, limitArg, sizeof(limitArg));
+	int limit = StringToInt(limitArg);
+
+	if(limit == 0)
+	{
+		limit = 16;
+		ReplyToCommand(client, "Setting limit failed, setting limits to 16 each");
+	}
+
+	g_Cvar_MaxRecons.SetInt(limit);
+	g_Cvar_MaxAssaults.SetInt(limit);
+	g_Cvar_MaxSupports.SetInt(limit);
```

Comment:
You could optionally use the `ConVar.GetBounds` method here to check whether this value can be used, and print an error if out of bounds.

Reply:
Yeah could do this, although, I think if we check the limit to be between 0 and MaxClients that should be fine since we already know these are the bounds for each ConVar?